### ISSUE TITLE
Redactor X handles script tags in html oddly

### ISSUE THREAD

**Issue by robinsowell** [ExpressionEngine/ExpressionEngine#4099]
So using Redactor X, you put a script tag in your html code, and view the html and it strips out the code from view, but it still saves the code.  So it's in the database, but it doesn't display in 'view code'.

I lean toward code should be allowed, but the 'view html' should show everything that is actually saved to the field/going to be saved to the field.

**Comment by rbroadford** [comment]
I recently came across this one as well :)

**Comment by ataucher** [comment]
Yes, this becomes a big problem when editing the same entry it will save without the original script tag in it.
In my context, I'm embedding forms from Wufoo with script tags but after one save they're removed. I'll have to hard code them into templates.
